check_cleaner_liveness_probe.sh

```diff
@@ -0,0 +1,17 @@
+PROBE_FILE="${LIVENESS_PROBE_FILE:-liveness_probe}"
+PROBE_STALE_MINUTES=$((3*"${EXPIRED_CLEANER_SLEEP_MINUTES:-20}"))
+
+if [ -e $PROBE_FILE ]; then
+    if [ $(find $PROBE_FILE -mmin +$PROBE_STALE_MINUTES) ]; then
```

Comment:
Missing quotes on all variables. I recommend to lint all sh files by shellcheck.

Reply:
Corrected in the updated code.